java/clients/src/main/java/sleeper/status/update/DownloadConfig.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2022-2023 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package sleeper.status.update;
+
+import com.amazonaws.services.s3.AmazonS3;
+import com.amazonaws.services.s3.AmazonS3ClientBuilder;
+import org.apache.commons.io.FileUtils;
+
+import sleeper.configuration.InstanceConfiguration;
+import sleeper.configuration.properties.table.TableProperties;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+import static sleeper.configuration.InstanceConfiguration.loadFromS3;
+import static sleeper.configuration.properties.table.TableProperty.DATA_BUCKET;
+import static sleeper.configuration.properties.table.TableProperty.TABLE_NAME;
+
+public class DownloadConfig {
+
+    private DownloadConfig() {
+    }
+
+    public static void main(String[] args) throws IOException {
+        if (args.length != 2) {
+            throw new IllegalArgumentException("Usage: <instance id> <directory to write to>");
+        }
+        String instanceId = args[0];
+        Path basePath = Path.of(args[1]);
+        clearDirectory(basePath);
```

Comment:
I wonder if we should get this to warn the user if the directory is not empty? It seems too easy to set the directory to . and for it to delete everything.

Reply:
Sure, we can have the scripts do the deletion instead of doing it here.